[00:17:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[00:17:22] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[00:17:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[00:17:22] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:00:33] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:00:33] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:00:33] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:00:33] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:10:29] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:10:29] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:43:40] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:43:40] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:43:51] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:43:51] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:53:39] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:53:39] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[02:27:06] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[02:27:06] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[02:27:07] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[02:27:07] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[02:36:53] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[02:36:53] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:00:17] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:00:17] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:10:27] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:10:27] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:10:28] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:10:28] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:20:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:20:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:43:40] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:43:40] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:53:53] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:53:53] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:53:53] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:53:53] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[04:03:50] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[04:03:50] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[04:27:24] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[04:27:24] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[04:37:39] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[04:37:39] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[04:37:39] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[04:37:39] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[04:47:30] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[04:47:30] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[05:11:00] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[05:11:00] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[05:21:19] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[05:21:19] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[05:21:19] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[05:21:19] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[05:31:12] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[05:31:12] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[05:54:48] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[05:54:48] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:05:04] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:05:04] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:05:05] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:05:05] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:15:01] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:15:01] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:38:44] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:38:44] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:49:10] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:49:10] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:49:10] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:49:10] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:59:07] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:59:07] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:22:48] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:22:48] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:33:06] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:33:06] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:33:07] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:33:07] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:43:03] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:43:03] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:55:39] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:55:39] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:06:59] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:06:59] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:17:19] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:17:19] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:17:20] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:17:20] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:26:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:26:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:39:50] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:39:50] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:51:05] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:51:05] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:01:09] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:01:09] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:01:20] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:01:20] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:01:32] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:01:32] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:01:43] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:01:43] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:01:54] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:01:54] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:02:05] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:02:05] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:02:16] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:02:16] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:02:27] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:02:27] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:02:38] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:02:38] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:02:49] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:02:49] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:03:00] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:03:00] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:03:11] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:03:11] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:03:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:03:22] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:03:33] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:03:33] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:03:44] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:03:44] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:03:55] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:03:55] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:04:06] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:04:06] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:04:16] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:04:16] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:04:27] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:04:27] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:04:38] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:04:38] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:04:50] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:04:50] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:05:01] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:05:01] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:05:12] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:05:12] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:05:23] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:05:23] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:05:34] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:05:34] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:05:45] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:05:45] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:05:55] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:05:55] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:06:06] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:06:06] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:06:17] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:06:17] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:06:28] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:06:28] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:06:39] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:06:39] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:06:50] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:06:50] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:07:01] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[09:07:01] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[09:07:24] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:07:24] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:07:24] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:07:24] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:09:29] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:09:29] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:22:59] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:22:59] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:35:21] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:35:21] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:51:10] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:51:10] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:51:11] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:51:11] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:53:13] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:53:13] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:06:44] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:06:44] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:19:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:19:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:35:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:35:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:35:19] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:35:19] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:37:23] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:37:23] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:50:56] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:50:56] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[11:03:17] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:03:17] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[11:19:19] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:19:19] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[11:19:20] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:19:20] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[11:21:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:21:22] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[11:34:58] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:34:58] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:01:51] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot4/dist/vkbot.js:72:35)
[14:01:51] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[14:35:21] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:35:21] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:19:13] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:19:13] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:02:24] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:02:24] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:46:21] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:46:21] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:29:54] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:29:54] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:13:56] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:13:56] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:57:52] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:57:52] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:40:58] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[19:40:58] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:24:54] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:24:54] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:08:44] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:08:44] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:52:44] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:52:44] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:36:43] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:36:43] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:20:44] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:20:44] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
